src/main/java/net/masterthought/cucumber/generators/AbstractPage.java

```diff
@@ -117,6 +118,13 @@ private void buildGeneralParameters() {
         context.put("build_number", configuration.getBuildNumber());
         context.put("directory_suffix", configuration.getDirectorySuffixWithSeparator());
 
+        context.put("js_files", configuration.getAdditionalJsFiles().stream()
```

Comment:
what if there is no custom files?

Reply:
I am not sure what you mean. If you do not add any css or js files to the Configuration? Then nothing changes.